pantheon-bundle/frontend/src/app/app.tsx

```diff
@@ -27,12 +27,16 @@ class App extends Component<any, IAppState> {
 
     const realFetch = self.fetch
     self.fetch = (input: RequestInfo, init?: RequestInit | undefined) => {
-      let newInput = input.toString()
-      if (window.location.host.startsWith('localhost') && input.toString().startsWith('/')) {
+      let newInput = input.toString();
+      if (
+        window.location.host.startsWith('localhost')
+        // Only changing requests Sling's old port
+        && parseInt(window.location.port) === 8080
+        && input.toString().startsWith('/')) {
```

Comment:
I do not recall why we originally needed this... are you sure it is still necessary? Did you try and remove it completely?

(I'm the one who wrote this method in the first place... I don't recall if it was for a frontend development workaround, or if it was so that our yarn tests would pass... if only I had included some comments about this...)

Reply:
It's required if you want it to work inside of Karaf. I'm not sure how all that stuff works, so I thought it'd be fine to leave it.

What's really weird is that the code is trying to reach `:8080` in Karaf, when it should be `:8181`, if we fix that then we can definitely get rid of this code. No idea what's causing it.

Reply:
I did some quick testing without the code and it doesn't look like it makes any difference anymore - so please take this opportunity to get rid of it.